### DISCUSSION TITLE
How to use cmd_vel with cyclonedds?

### DISCUSSION THREAD

**Discussion by Juancams** [abizovnuralem/go2_ros2_sdk#50] [Q&A]
Hii,

First of all, thank you very much for supporting this package, it is being very helpful to me.

I have been trying to move the go2 using the webrtc connection mode, and also using navigation2 to move it, and everything works correctly (although the rotation reads it from linear z and not from angular z). How could I make it work with cyclonedds?

Thank you very much^^

**Comment by abizovnuralem** [comment]
Currently, this repo does have full support for cycloneDDS (It is under dev now), you can see the topics from there and write your code to control the dog, the idea is the same as for webrtc

**Comment by Juancams** [comment]
Hello good,

I tell you that we are doing work in parallel, in this case in C++ and only to support cyclonedds, where we have already supported the cmd_vel that I mentioned. I leave you the [repository](https://github.com/IntelligentRoboticsLabs/go2_robot/tree/humble-devel) here.

I hope there is no problem with this, and that we can also at some point coordinate our work to improve both the c++ and python implementations.

**Comment by abizovnuralem** [comment]
Great! Will see it
